backend/django/core/utils/util.py

```diff
@@ -205,6 +209,27 @@ def create_metadata_objects(df, project):
         MetaData.objects.bulk_create(metadata_objects)
 
 
+def create_label_similarity_results(project):
```

Comment:
This looks great! One thing I forgot before though, if someone adds data to the database, right now this is going to run on all of the data, including the parts which were already created.

I think you should be able to filter the Data objects by those that don't have any DataLabelSimilarityPairs objects, but let me know if that causes any trouble

Reply:
This comment is probably resolved, but a moot point since we're moving the suggestions to another MR